js/main.js

```diff
@@ -1,13 +1,223 @@
 /* global _ */
-var content = "";
-var error = "";
 var xhr = new XMLHttpRequest();
 
-// construct a quote
+var singularize = function(word) {
+    word = word.trim();
+    if (word.substr(-1) === 's') {
+        return word.slice(0, -1);
+    } else {
+        return word;
+    }
+};
+
+//
+// Things Warren Ellis Calls "My Little X"
+//
+var ellisMyLittleLabel = function(vocabulary) {
```

Comment:
Good feedback. I did a little reading on that, and basically stopped when I saw "Doesn't work in IE9", though I suppose that's no longer a good reason to avoid something. For now, I'll leave it like this.